evap/staff/templates/staff_course_review.html

```diff
@@ -4,13 +4,14 @@
 
 {% block content %}
     {{ block.super }}
-    <form action="{% url "evap.staff.views.course_review" semester.id course.id offset %}" method="POST" class="form-horizontal multiselect-form">
+    <form action="{% url "evap.staff.views.course_review" semester.id course.id %}" method="POST" class="form-horizontal multiselect-form">
     <fieldset>
         <legend>{% trans "Answer Review" %}</legend>
         <p>{% blocktrans with TextAnswer.elements_per_page as elements_per_page and course.open_textanswer_set|length as open and course.textanswer_set|length as total %}Note: For better usability only {{ elements_per_page }} comments are shown at once. For this course {{ open }} out of {{ total }} comments have not been reviewed.{% endblocktrans %}</p>
 
         {% csrf_token %}
         {{ formset.management_form }}
+        <input type="hidden" name="skipped_answers" value="{{ skipped_answers }}" id="skipped_answers" />
```

Comment:
i'm still a little bit confused. do you need this hidden field in the form and the url parameters?